libp2p/crypto/interface.go

```diff
@@ -0,0 +1,15 @@
+package crypto
+
+import (
+	coreCrypto "github.com/libp2p/go-libp2p/core/crypto"
+	"github.com/multiversx/mx-chain-core-go/core"
+	"github.com/multiversx/mx-chain-crypto-go"
+)
+
+// P2PKeyConverter defines what a p2p key converter can do
+type P2PKeyConverter interface {
+	ConvertPrivateKeyToLibp2pPrivateKey(privateKey crypto.PrivateKey) (coreCrypto.PrivKey, error)
```

Comment:
we need to remove coreCrypto as this will propagate in all other repos.
Maybe leave out this function?
Then we can safely move this interface definition in the mx-chain-p2p-go/interface.go

Reply:
pushed